src/test/java/org/zendesk/client/v2/UserTest.java

```diff
@@ -1,20 +1,19 @@
 package org.zendesk.client.v2;
 
-import static com.github.tomakehurst.wiremock.client.WireMock.equalTo;
-import static com.github.tomakehurst.wiremock.client.WireMock.equalToJson;
-import static com.github.tomakehurst.wiremock.client.WireMock.ok;
-import static com.github.tomakehurst.wiremock.client.WireMock.put;
-import static com.github.tomakehurst.wiremock.client.WireMock.putRequestedFor;
-import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
-import static com.github.tomakehurst.wiremock.client.WireMock.urlPathEqualTo;
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
```

Comment:
Nit: I'd rather see the explicit import kept (we strongly discourage using wildcard imports even in tests, see https://github.com/cloudbees-oss/zendesk-java-client/blob/master/CONTRIBUTING.adoc#imports).

(sorry I just noticed the project doesn't have a working editorconfig file to help setting up your IDE, I'll try to add it at some point).

Reply:
Good spot - I'll get that corrected

Reply:
@PierreBtz - I'll be creating a local .editorconfig for this repo (to make sure that I conform to the contributing guidelines when letting IntelliJ do it's thing).
If you want, I could submit that in a separate PR

Reply:
Works for me :)
Thanks!